package.json

```diff
@@ -2,7 +2,7 @@
   "name": "jade-data-repo-ui",
   "title": "jade-data-repo-ui",
   "description": "Data Repository UI",
-  "version": "0.4.0",
+  "version": "0.7.0",
```

Comment:
These versions were bumped during my tests. We need to merge this to keep all of versions in sync.